build_node/build_node_builder.py

```diff
@@ -129,22 +134,27 @@ def run(self):
                 try:
                     build_artifacts = self.__upload_artifacts(
                         artifacts_dir, task_log_file, only_logs=only_logs)
-                except Exception as e:
-                    self.__logger.exception('Cannot upload task artifacts: %s',
-                                            str(e))
+                except Exception:
+                    self.__logger.exception('Cannot upload task artifacts:')
```

Comment:
The colon is no longer required if you're not going to print anything after it

Reply:
Looks like copy-paste text, because I saw that in other places